src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -266,6 +275,39 @@
   }
 }
 
+.participant__join-icon,
+.participant__kick-icon {
+  width: $icon--large;
+  height: $icon--large;
+  padding: 0;
+  margin-left: auto; // align to the very right
+  border-radius: $note__border-radius;
+  border: none;
+  background-color: $color-white;
+  width: $icon--large;
+  height: $icon--large;
+  flex-shrink: 0;
+
+  cursor: pointer;
+
+  &--active {
+    background-color: var(--accent-color);
+    color: $color-white;
+  }
+  &:focus {
+    outline: 2px solid var(--accent-color);
+  }
+
+  &:hover {
+    background-color: var(--accent-color--desaturated-light);
+  }
+}
+
+.participants__ban-dialog .confirmation-dialog__icon {
+  width: 77px;
+  height: 77px;
```

Comment:
77px seems oddly specific 😄 Where does this number come from?

Reply:
It was in there before and comes from the size in the design on Figma.
I merely removed it from the svg and moved it into the css.